packages/block-library/src/table-of-contents/ListItem.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+
+export default function ListItem( props ) {
+	const { children, noWrapList = false } = props;
+	let childNodes = null;
+
+	if ( children ) {
+		childNodes = children.map( function( childNode, index ) {
```

Comment:
`childNodes` isn't used except inside the `if` statement, and it's never reassigned.
```suggestion

	if ( children ) {
		const childNodes = children.map( function( childNode, index ) {
```